umapi_client/functional.py

```diff
@@ -195,15 +145,12 @@ def update(self, email=None, username=None, first_name=None, last_name=None, cou
         :param country: new country for this user
         :return: the User, so you can do User(...).update(...).add_to_groups(...)
         """
-        if email:
-            self._validate(email=email)
-        if username:
-            self._validate(username=username)
         updates = {}
         for k, v in six.iteritems(dict(email=email, username=username,
                                        firstname=first_name, lastname=last_name,
                                        country=country)):
-            if v: updates[k] = v
+            if v:
+                updates[k] = v
```

Comment:
Normally I would chastise you for introducing a formatting change in the context of a functional change (rather than as a separate commit).  But what the heck, this is small enough not to worry about it.

Reply:
I'm all for PEP8 compliance! ;) -d.